devel/xcodes/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup 		RobotsAndPencils xcodes 0.16.0
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+homepage			https://github.com/RobotsAndPencils/xcodes
```

Comment:
The GitHub portgroup will set the `homepage` to this value. So this redundant line should be removed.